juju/juju
# Introduce a new JUJU_FEATURES env var for optin to new features.

## Description of change

To enable users to opt in to using new, documented Juju features which cannot be enabled by default, add a new JUJU_FEATURES flag. Such features may not be enabled out of the box due to the requirement to maintain compatibility.

This flag is client only, so users can turn it on as needed - they can leave it off to continue to run scripts they don't want to update yet, but turn it on for their interactive use to get the benefit of any new CLI changes etc.

## QA steps

export JUJU_FEATURES=actions-v2

check that the new actions UX is enabled, juju run, juju exec, juju cancel-action etc
